docs/cryptography/playfair-cipher.mdx

```diff
@@ -0,0 +1,503 @@
+    the decrypted pair for every pair in diagraph based on the positions of the chars.
+    :param digraphs: 2D list containing the encrypted text in the form of pairs
+    :type list
+    :param matrix: 2D list with 25 english alphabets formed using the key
+    :type list
+    :return encrypted_text: the decrypted form of encrypted text
+    :type str
+    '''
+    encrypted_text = ''
+    for pair in digraphs:
+        # finding the location of chars in digraph pairs in key matrix
+        first, last = pair
+        first_row, first_col = get_location(first, matrix)
+        last_row, last_col = get_location(last, matrix)
+        by = ''
+        # if both the chars in the pair are in same row of key matrix
+        if first_row == last_row:
+            en_first_col = get_new_pos(first_col)
+            en_last_col = get_new_pos(last_col)
+            by = 'col'
+        # if both the chars in the pair are in the same column of key matrix
+        elif first_col == last_col:
+            en_first_row = get_new_pos(first_row)
+            en_last_row = get_new_pos(last_row)
+            by = 'row'
+        # if both the row and column for the chars in the pair are different
+        else:
+            en_first_col = last_col
+            en_last_col = first_col
+            by = 'col'
+        # find the decrypted text based on the new char positions
+        if by == 'col':
+            encrypted_text += matrix[first_row][en_first_col]
+            encrypted_text += matrix[last_row][en_last_col]
+        elif by == 'row':
+            encrypted_text += matrix[en_first_row][first_col]
+            encrypted_text += matrix[en_last_row][last_col]
+
+    return encrypted_text
+
+
+def main():
+
+        option = int(input('Enter 1 for encrypting and 2 for decrypting : '))
+        # Input for encryption task
+        if option == 1:
+            key = input('Enter the key : ')
+            key_matrix = create_matrix(key)
+            text = input("Enter the plain text for encryption: ")
+            plain_text = clean_text(text)
+            digraphs = create_digraphs(plain_text)
+            encrypted_text = encrypt(digraphs, key_matrix)
+            print("Your Encrypted text is ",encrypted_text)
+
+        # Input for decryption task
+        elif option == 2:
+            key = input('Enter the key : ')
+            key_matrix = create_matrix(key)
+            en_text = input("Enter the encrypted text for decryption: ")
+            digraphs = create_digraphs(en_text)
+            decrypted_text = decrypt(digraphs, key_matrix)
+            print("Your Decrypted text is ", decrypted_text)
+
+if __name__ == '__main__':
+    main()
+```
+
+</TabItem>
+</Tabs>
+
+## Sample Input and Output
+
+### Input:
+
+```
+Enter the key : hello
+Enter the plain text for encryption: world
+```
+
+### Output:
+
+```
+Your Encrypted text is kbekdv
+```
+
+## Complexity Analysis
+
+```
+Time  complexity - O(n+m)
+Space complexity -O(n+m)
+```
```

Comment:
No need to keep them in triple backticks. Make them bullet points and add the values in backticks